---a/net/minecraft/item/ItemCoal.java
+++b/net/minecraft/item/ItemCoal.java
@@ -1,23 +1,18 @@
 package net.minecraft.item;
 
-import java.util.List;
 import net.minecraft.creativetab.CreativeTabs;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class ItemCoal extends Item
-{
+
+public class ItemCoal extends Item {
+
     private static final String __OBFID = "CL_00000002";
 
-    public ItemCoal()
-    {
+    public ItemCoal() {
         this.setHasSubtypes(true);
         this.setMaxDamage(0);
         this.setCreativeTab(CreativeTabs.tabMaterials);
     }
 
-    public String getUnlocalizedName(ItemStack stack)
-    {
+    public String getUnlocalizedName(ItemStack stack) {
         return stack.getMetadata() == 1 ? "item.charcoal" : "item.coal";
     }
-
-    @SideOnly(Side.CLIENT)
-    public void getSubItems(Item itemIn, CreativeTabs tab, List subItems)
-    {
-        subItems.add(new ItemStack(itemIn, 1, 0));
-        subItems.add(new ItemStack(itemIn, 1, 1));
-    }
-}
+}
